File: problems/HackerRank/2521_exercises/tree/postOrder.c
void postOrder(struct node *root)
{
	if (!root)
	{
		return;
	}

	if (root->left)
	{
		postOrder(root->left);
	}
	if (root->right)
	{
		postOrder(root->right);
	}
	printf("%d ", root->data);
}

File: problems/HackerRank/LeetCode/Easy/postorder.py
class Solution:
    def postorder(self, root: 'Node') -> List[int]:
        queue = []

        def dfs(node):
            if(node == None):
                return
            
            for child in node.children:
                dfs(child)
            queue.append(node.val)
        
        dfs(root)
        return queue
        